sigs.yaml

```diff
@@ -437,6 +437,11 @@ sigs:
       - https://raw.githubusercontent.com/kubernetes/kubernetes/master/pkg/kubelet/token/OWNERS
       - https://raw.githubusercontent.com/kubernetes/kubernetes/master/pkg/serviceaccount/OWNERS
       - https://raw.githubusercontent.com/kubernetes/kubernetes/master/plugin/pkg/admission/serviceaccount/OWNERS
+    - name: multi-tenancy
+      description: >
+       Multi-tenancy related proposals and prototypes.
```

Comment:
Can you be a bit more specific here? For instance, adding a new authorization feature that enables multitenant usecases would fall under the authorizers subproject, even if it's "multi-tenancy related".

Reply:
I'll leave it to @easeway and @davidopp to come up with a proper description. :)

Reply:
My proposal is: _proposals and prototypes for introducing tenant model to enable multi-tenant cluster_

Reply:
Done!